config/envoyconfig/testdata/main_http_connection_manager_filter.json

```diff
@@ -19,6 +19,10 @@
         }
       }
     ],
+    "accessLogOptions": {
+      "flushAccessLogOnNewRequest": true,
```

Comment:
I think this results in logging each request twice, once when the request is received and again after it has been processed. For example, if I make one request with `curl` I see two access log entries with the same request ID:
```json
{
  "level": "info",
  "service": "envoy",
  "upstream-cluster": "",
  "method": "GET",
  "authority": "verify.localhost.pomerium.io",
  "path": "/",
  "user-agent": "curl/8.7.1",
  "referer": "",
  "forwarded-for": "192.168.0.77",
  "request-id": "e6d64ad4-7515-4c4d-8493-63a80ac34796",
  "duration": 0,
  "size": 0,
  "response-code": 0,
  "response-code-details": "",
  "time": "2024-08-07T11:08:08-07:00",
  "message": "http-request"
}
{
  "level": "info",
  "service": "envoy",
  "upstream-cluster": "route-669ed162c9a9d706",
  "method": "GET",
  "authority": "localhost:8000",
  "path": "/",
  "user-agent": "curl/8.7.1",
  "referer": "",
  "forwarded-for": "192.168.0.77",
  "request-id": "e6d64ad4-7515-4c4d-8493-63a80ac34796",
  "duration": 5.736167,
  "size": 446,
  "response-code": 200,
  "response-code-details": "via_upstream",
  "time": "2024-08-07T11:08:08-07:00",
  "message": "http-request"
}
```

I'd suggest we keep the existing access log options for now.